pom.xml

```diff
@@ -58,7 +58,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <liquibase.version>0-SNAPSHOT</liquibase.version>
+        <liquibase.version>4.18.0</liquibase.version>
```

Comment:
Should we not go to the latest version of Liquibase which was released last week i.e. `4.19.0`?

Reply:
@KushnirykOleh -- could you check on this too please.